dokan-wpml.php

```diff
@@ -1005,16 +1031,25 @@ public function vendor_subscription_pack_title_translation( $title, $product ) {
      * @return mixed
      */
     public function get_raw_option( $option, $section, $default = '' ) {
+        $cache_key = $section . '_' . $option;
+
+        if ( isset( $this->cached_options[$cache_key] ) ) {
```

Comment:
Please follow the WordPress coding standard guidelines in whole PR. [Link](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#whitespace)